src/platform/site-wide/sass/minimal.scss

```diff
@@ -0,0 +1,60 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
I've made this file to be a subset of the styles included in [the site-wide stylesheet](https://github.com/department-of-veterans-affairs/vets-website/blob/1229bcfcd559435bcce3d23c8bf899bbb6650bd3/src/platform/site-wide/sass/style.scss). Notable changes include:

- importing pieces of USWDS, fontawesome, and base formation styles instead of formation's `core` module
- excluding most of formation's sass modules (i.e. `m-additional-info`)
- excluding the facilities sass modules

As the migration away from formation continues we should be able to minimize this even further.